src/Features/CSharp/Portable/CodeFixes/AddImport/CSharpAddImportCodeFixProvider.cs

```diff
@@ -700,7 +701,11 @@ private bool IsViablePropertyOrField(ISymbol propertyOrField, SyntaxNode express
                 return false;
             }
 
-            return string.Compare(propertyOrField.ContainingType.Name, leftName.Identifier.Text, this.IgnoreCase) == 0;
+            var comparisonType = this.IgnoreCase
+                ? StringComparison.OrdinalIgnoreCase
```

Comment:
Use Microsoft.CodeAnalysis.CaseInsensitiveComparison to get all that exciting Turkish "i" special casing.